lib/dirList.js

```diff
@@ -159,10 +160,7 @@ const dirList = {
    * @return {boolean}
    */
   handle: function (route, options) {
-    if (!options.names) {
-      return false
-    }
-    return options.names.includes(path.basename(route)) ||
+    return options.names?.includes(path.basename(route)) ||
       // match trailing slash
       (options.names.includes('/') && route[route.length - 1] === '/')
```

Comment:
```suggestion
      (options.names?.includes('/') && route[route.length - 1] === '/')
```

Without this change, this will throw if `options.names` is not defined, and the whole function will return `undefined` instead of `false`, so its signature needs to be updated as well

Reply:
Good comment. Fixed!